cashback/README.md

```diff
@@ -1,40 +1,44 @@
 # Cashback
 
-This is a simple use-case application created using lisk-sdk@alpha.
+This is a simple use-case application created using alpha version of `lisk-sdk`.
 
-Casback application extends the Transfer Transaction of SDK by adding an additional "cashback" feature - with every balance transfer. 10% of transferred amount goes back to the sender's account increasing the total tokens' supply in the network.
+Casback application extends the Transfer Transaction of SDK by adding an additional `cashback` feature - with every balance transfer, 10% of transferred amount goes back to the sender's account increasing the total tokens' supply in the network.
 
 ## Usage
 
-Please visit our [comprehensive tutorial](https://github.com/LiskHQ/lisk-docs/blob/master/start/tutorials/cashback.md) for a full explanation of how to build this example.
+Please visit our [comprehensive tutorial](https://github.com/LiskHQ/lisk-docs/blob/master/start/tutorials/cashback.md) for a full explanation of how to build this example step by step.
```

Comment:
Better link here to the documentation website https://lisk.io/documentation/start/tutorials/cashback

Reply:
I belive we already have the link above for `comprehensive tutorial`, do we need it for `step by step`?

Reply:
The link above is leading to Github, I suggest to lead directly to our documentation wbesite, and not on the docs-pages on Github :)